test/javascript/commission_histories/index.test.js

```diff
@@ -21,19 +21,16 @@ var mock_commission_changes = function(){
   });
 }
 
-
 describe('index_template', ()=> {
   it('has correct params without query', ()=>{
     Vue.component('BPagination', BPagination)
     mock_commission_changes()
     const wrapper = shallowMount(IndexTemplate, {
-      propsData: {
-        network: 'testnet'
-      },
+      store,
       stubs: {BPagination: BPagination}
     })
 
-    expect(wrapper.vm.api_url).toMatch("/api/v1/commission-changes/testnet?")
+    expect(wrapper.vm.api_url).toMatch("/api/v1/commission-changes/mainnet?")
```

Comment:
Why do we change to mainnet in tests?

Reply:
network comes from Vuex's getter. I managed to set this up in recent commit.